Assets/AirshipPackages/@Easy/Core/Shared/Character/CharacterRendering.ts

```diff
@@ -74,7 +74,15 @@ export default class CharacterRendering extends AirshipBehaviour {
 		this.cameraBin.Clean();
 
 		if ((this.character.IsInitialized() && this.character?.IsLocalCharacter()) || this.enableIfNotLocalCharacter) {
-			const headTransform = this.character.rig.head;
+			let headTransform = this.character.rig.head;
+			if (!headTransform) {
+				warn(
+					"Unable to find transform for character. Add a transform to the character rigs head to use CharacterRendering transparency or delete CharacterRendering from your character.",
```

Comment:
Change to this:

[CharacterRendering]: Missing head transform on character rig. Add a head transform to character rig or delete CharacterRendering from your character.